### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by LoveYao** [SudTechnology/hello-sud-plus-ios#1]
Is the game module implemented by H5? Can this pass the review of the app store?

**Comment by kaniel520** [comment]
Yeah, Off cause It can be published to the app store, It's implemented by native view, not the H5.
